fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java

```diff
@@ -180,9 +182,12 @@ private void handleQuery() {
 
         // execute this query.
         StatementBase parsedStmt = null;
+        List<Pair<StatementBase, PQueryStatistics>> auditInfoList = Lists.newArrayList();
+        boolean alreadyAddedToAuditInfoList = false;
```

Comment:
`alreadyAddedToAuditInfoList` seems useless, we can just check `auditInfoList.isEmpty()`

Reply:
if some statement execute failed before added to auditInfoList， we should try to record it